gfsphysics/GFS_layer/GFS_physics_driver.F90

```diff
@@ -4949,32 +4949,30 @@ subroutine GFS_physics_driver                                 &
 !       write(1000+me,*)' maxwatncb=',maxval(Stateout%gq0(1:im,k,ntlnc)),' k=',k,' kdt',kdt
 !       enddo
 
-!## CCPP ##* m_micro.F90/m_micro_run
```

Comment:
This comment in line 4952 and in line 4977 in the "old" code should remain. I can add it back in as part of our next NCAR dtc/develop to EMC develop PR.

Reply:
Made this change in NCAR's dtc/develop branch. Will come to EMC develop with the next commit from ourside.